### DISCUSSION TITLE
databend-query：can't detect region automatically, unexpect status code got

### DISCUSSION THREAD

**Discussion by lddlww** [databendlabs/databend#9663] [Q&A] [answered]
```
databend-query --config-file=/data/databend/9000/config/databend-query.toml
Databend Query start failure, cause: Code: 4000, displayText = other error (Unexpected (permanent) at , context: { status: 502 } => can't detect region automatically, unexpect status code got).
```
what's wrong with it?

version: ```databend-query v0.9.5-nightly-6772df6(rust-1.68.0-nightly-2023-01-17T16:27:18.567251878Z)```

**Comment by sundy-li** [comment]
Thanks, it's a bug, can you get the backtrace now? 

`RUST_BACKTRACE=1  databend-query --config-file=/data/databend/9000/config/databend-query.toml`

**Comment by sundy-li** [comment, marked as answer]
seems related to opendal's s3 region auto detection  @Xuanwo

**Reply by lddlww** [reply]
thanks，i found my minio sidekick endpoint config error，databend-query run normal now
